Polygon_mesh_processing/include/CGAL/Rigid_triangle_mesh_collision_detection.h

```diff
@@ -176,6 +176,25 @@ class Rigid_triangle_mesh_collision_detection
     : m_free_id(0)
   {}
 
+  /**
+    * Constructor moving an instance of Rigid_triangle_mesh_collision_detection to a new memory
+    * location with minimal memory copy.
+    * @param other The instance to be moved
+    */
```

Comment:
Just say "move-constructor", because the documentation you gave is at least imprecise, and maybe incorrect.